katello/hooks/pre_commit/13-hiera.rb

```diff
@@ -29,14 +33,22 @@
   # Check if it's actually 90% of the required. If a crash kernel is enabled
   # then the reported total memory is lower than in reality.
   if facts[:memory][:system][:total_bytes] < (required_memory * 1024 * 1024 * 1024 * 0.9)
-    say "<%= color('Insufficient memory for tuning size', :bad) %>"
-    say "Tuning profile '#{new_tuning}' requires at least #{required_memory} GB of memory and #{required_cores} CPU cores"
+    if app_value(:tuning)
+      say "<%= color('Insufficient memory for tuning size', :bad) %>"
+      say "Tuning profile '#{new_tuning}' requires at least #{required_memory} GB of memory and #{required_cores} CPU cores"
+    else
+      say "This scenario requires at least #{required_memory} GB of memory and #{required_cores} CPU cores"
```

Comment:
There's [scenario_id](https://github.com/theforeman/kafo/blob/06187fab9efb3ad2ef9708e5f2c9cea7501168c1/lib/kafo/hook_context.rb#L110-L113) as well if you want to use it. This is the same value as passed to `--scenario`, so `foreman`, `katello` or `foreman-proxy-content`.

Reply:
good idea, updated now.

Reply:
the new output looks like:
```console
# foreman-installer --scenario foreman-proxy-content
puppet-agent-oauth-0.5.1-3.el7.noarch
This scenario 'foreman-proxy-content' requires at least 8 GB of memory and 1 CPU cores
```